tests/Doctrine/Tests/Models/EagerBatched/Tag.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: wouter
+ * Date: 11/18/15
+ * Time: 1:00 PM
+ */
+
+namespace Doctrine\Tests\Models\EagerBatched;
+
+/**
+ * Class Tag
+ * @package Doctrine\Tests\Models\EagerBatched
+ *
+ * @Entity()
+ * @Table(name="eager_batched_tag")
+ */
+class Tag
+{
+
+	/**
+	 * @Column(type="string")
+	 * @Id()
+	 */
+	protected $name;
+
+	public static function factory(Article $a, $tag) {
+		$me = new static();
+		$me->setArticle($a);
+		$me->setName($tag);
+		return $me;
+	}
+
+	/**
+	 * @ManyToOne(targetEntity="Article",cascade={"persist"},fetch="LAZY",inversedBy="tags")
+	 * @Id()
+	 */
+	protected $article;
```

Comment:
I don't use it a lot in production code either, but in this case it does make the test code a bit cleaner.